lib/request.js

```diff
@@ -292,8 +291,7 @@ module.exports = {
    */
 
   get idempotent() {
-    const methods = ['GET', 'HEAD', 'PUT', 'DELETE', 'OPTIONS', 'TRACE'];
-    return !!~methods.indexOf(this.method);
+    return ['GET', 'HEAD', 'PUT', 'DELETE', 'OPTIONS', 'TRACE'].includes(this.method);
```

Comment:
Why would it drop performance? I don't see it making any difference in that regard. Since it's a global it might actually allow *more* optimizations, IDK.